### DISCUSSION TITLE
Couldn't start client Flow language server

### DISCUSSION THREAD

**Discussion by kasperh1-coder** [area9innovation/flow9#971] [Q&A] [answered]
Hi, 

I am trying to get started with Flow. But when running the demos.flow file in vs code I get the following error message:

![image](https://user-images.githubusercontent.com/99254844/152973104-9a76f871-1cef-4a60-8d2b-331003ff95be.png)

Any tips to fix this issue?

**Comment by alstrup** [comment, marked as answer]
Do you have java runtime installed? Did you configure the flow extension paths? Can you add flow9/bin to your path and see what "flowc1" does?
